src/Microsoft.Graph/Models/Extensions/ContentConverterExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Graph.Extensions
+{
+    /// <summary>
+    /// Implements Content Converter Extensions
+    /// </summary>
+    public static class ContentConverterExtensions
```

Comment:
I suggest we call this DateTimeZoneExtensions as that is the class we are extending

Reply:
@MIchaelMainer and I had a discussion on this and we settled for the name ContentConverterExtensions given that going forward all extensions method will be implemented in this class.

Reply:
Not all extensions. But we expect that we will need to convert more data types in the future.